app/routes/events/view.js

```diff
@@ -9,7 +9,7 @@ export default Route.extend({
 
   model(params) {
     return this.store.findRecord('event', params.event_id, {
-      include: 'event-topic,event-sub-topic,event-type,event-copyright,tax,sponsors'
+      include: 'event-topic,event-sub-topic,event-type,event-copyright,tax,sponsors,speakers-call,microlocations,tracks,session-types'
```

Comment:
Like for example, load session types, speaker calls, tracks, micro-locations, custom forms only for wizard step 3, sponsors only for wizard step 2, etc